wiki/Beatmapping/Mapping_techniques/Rhythm/en.md

```diff
@@ -165,8 +165,7 @@ These consist of sliders (with or without repeats), hit circles or a combination
 
 The job of a deathstream is to practically fail players that cannot handle ordinary streams.
 
-The use of deathstreams will make your map **unrankable**!
-However, it may go through as one of the *approved* beatmaps.
+The use of deathstreams used to make your map unrankable, but now it is rankable, as approved category no longer exists.
```

Comment:
"deathstreams" have never been unrankable/strictly disallowed by the Ranking Criteria in practice tho

it's just that back then when osu! was still in its Score Ranking era (and Ranked beatmaps were prohibited from having 20mil+ max score), the usage of super long streams were rather discouraged as they tend to crank up the max score rapidly

Reply:
Yeah, I just want to update this, as deathstreams are common in ranked maps.

Reply:
might wanna word this as such

```suggestion
During the [score ranking](/wiki/Ranking#score-ranking) era, the usage of deathstreams were strictly limited as the [Ranking Criteria](/wiki/Ranking_Criteria) would only allow for a Ranked beatmap to have a maximum of 20 million Ranked score back then.
```